index.js

```diff
@@ -589,11 +619,32 @@ Peer.prototype._createOffer = function () {
   }).catch(function (err) { self.destroy(makeError(err, 'ERR_CREATE_OFFER')) })
 }
 
+Peer.prototype._requestMissingTransceivers = function () {
+  var self = this
+
+  ;['audio', 'video'].forEach(kind => {
```

Comment:
In case we already use ES6 features (like arrow functions), we might want to also use nicer loops:
```suggestion
  for (const kind of ['audio', 'video']) {
```